.github/scripts/copywrite.sh

```diff
@@ -4,6 +4,9 @@
 # Checks if a file has the correct copywrite
 #
 
+# Ignore generated files where we cannot use a template
+grep -q "$1" .github/scripts/excluded_from_copywrite && exit 0
```

Comment:
nit: not for this PR but it's copyright... or is this a play on writing the copyright?

Reply:
The script is called copywrite.sh so I just used the same name. But agree on using the right name everywhere

Reply:
Will do the cleanup in a subsequent PR